jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py

```diff
@@ -190,74 +183,107 @@ def infer(
         )
 
 
+def training(use_mlflow, model, training_parameters, dataset, train_data):
+    model.train()
+    if training_parameters is not None:
+        with open(t_config_fname, "w+") as jsonfile:
+            train_config.update(training_parameters)
+            json.dump(train_config, jsonfile, indent=4)
+    for data in dataset.keys():
+        for dat in dataset[data]:
+            train_data["contexts"].append(dat)
+            train_data["candidates"].append(data)
+            train_data["labels"].append(1)
+    model = train_model(
+        use_mlflow=use_mlflow,
+        model=model,
+        tokenizer=tokenizer,
+        contexts=train_data["contexts"],
+        candidates=train_data["candidates"],
+        labels=train_data["labels"],
+        train_config=train_config,
+    )
+    return model
+
+
 # API for training
 @jaseci_action(act_group=["bi_enc"], allow_remote=True)
-def train(dataset: Dict = None, from_scratch=False, training_parameters: Dict = None):
+def train(
+    use_mlflow: bool = True,
```

Comment:
We have to make use_mlflow =false

Reply:
okay are you making this change or are you expecting me to?

Reply:
this is from satyam's repository, not sure how to update this